handbook.md

```diff
@@ -0,0 +1,233 @@
+
+Be mindful of the use of the word 'you'. It can be easily misunderstood as you reviewing the 'person' not the 'code'. Try not to personalize any references to code. I.e: 'you changed that in line 42' vs 'that change in line 42'.
+
+Back up your reviews by facts when they are not obvious or when contributors might not know why they should make some change. Adopt a "teaching" approach, but again, back up your 'teachings' with links to other sources.
+
+Use '@' to raise the attention of the contributor about any issue. If an user is not 'watching' the project on GitHub or is not subscribed to a particular issue, GitHub will not show notifications nor send emails to that user.
+
+If there is conflict, point to this handbook for reference.
+
+#### Checklist
+http://projects.theforeman.org/projects/foreman/wiki/Reviewing_patches
+
+#### Labels
+
+Labels help reviewers understand what is the status of a pull request. Thanks to them, reviewers can make sure no pull request remains unreviewed and when they get updated, they see it.
+
+* PR processor will set the labels `Not reviewed` and `Needs testing` when a pull request has just been submitted.
+* After you have reviewed the pull request, mark it as 'Waiting on contributor' if it needs changes or you're expecting answers.
+* When the pull request branch gets updated, PR processor will automatically set the labels 'Needs re-review' and 'Needs testing'.
+* Repeat the process until the code is ready to be merged.
+* If there is a problem such as a serious disagreement between people about a change, or there's a dependency on other project that needs to be solved, set it as 'Reached an impasse'.
+
+## Becoming a maintainer
+
+As a maintainer, your responsibility is to keep a project running, taking care of certain parts of it. Examples of such parts are:
+* A plugin, by taking care of documentation, code reviews, and packages to foreman-packaging
+* Debian packaging, by looking at which projects need debian packages and updating them
+* A subsystem, such as a Compute Resource or an Operating System in Foreman and updating it properly.
+
+Traditionally, the process to become a maintainer has been simple. Take care of something for long enough and people will expect you to maintain it. We might ask you directly if you want to be a maintainer if you're maintaining something for some time. If you have made contributions only on a very specific topic (an Operating System, for instance), we might ask you to be a maintainer for that part of the code.
+
+## API usability and versioning
+
+We follow [Semantic versioning 2.0](http://semver.org/) for anything above '0.x' releases. This means incompatible changes, such as breaking an API endpoint, not working with a previous Foreman version (in the case of a plugin), require increasing the major version. The bulk of your changes should only require increasing the minor and patch versions, for backwards-compatible functionality and bugfixes.
```

Comment:
Yes, I usually mean core to mean the four projects foreman, smart-proxy, foreman-installer, foreman-selinux which are managed and released together.